mitodl/mitxonline
# Catalog page css updates - v1

# What are the relevant tickets?
https://github.com/mitodl/hq/issues/1676

# Description:
This PR cleans up some of the extraneous CSS on the catalog page.  
The only visible changes this PR makes are:
1. Setting the font family.
2. Slightly decreasing the width of the department filter sidebar and catalog items.

# How can this be tested?

1. Visit http://mitxonline.odl.local:8013/catalog/.
2. Ensure that the page renders correctly for all screen sizes.
3. Ensure that the page is accessible via screen readers and can be navigated with a keyboard.
